src/common/xo/index.js

```diff
@@ -41,6 +41,10 @@ export const XEN_DEFAULT_CPU_CAP = 0
 
 // ===================================================================
 
+export const XEN_VGA_VIDEORAM_MAX = 16
```

Comment:
BTW, are you sure this is the maximum value?

Provide a link to the documentation.